lib/index.js

```diff
@@ -92,7 +92,7 @@ module.exports = Adapter => class PostgreSQLAdapter extends Adapter {
       this.pool = new Pool(connection)
     }
 
-    // Set `client` alias for backwards compatibility.
+    // This is overridden with a single client instance during a transaction.
     this.client = this.pool
```

Comment:
I'm not sure if any of the changes below affect anything, since `this.client` is effectively an alias for `this.pool`.

Reply:
Although initially it's an alias for `this.pool`, during a transaction it's overridden with a client instance. See `lib/index.js:562` where it's replaced in the scope with `Object.assign()`.

Now that the CRUD methods are directly using `this.client` it's no longer just for backwards compatibility.